src/common/paths/tasks_taskID_runs_runID_retry.yml

```
@@ -3,6 +3,11 @@ post:
   tags:
     - Tasks
   summary: Retry a task run
+  requestBody:
+    content:
+      application/json:
```

I'd also consider setting the charset (seems some content type checks are more pedantic than others, annotations i believe currently is)
```suggestion
      application/json; charset=utf-8:
```

```diff
--- orig-ts/common.ts
+++ ts/common.ts
@@ -7365,6 +7365,8 @@
   taskID: string
   runID: string
 
+  data?: any
+
   headers?: {
     'Zap-Trace-Span'?: string
   }
@@ -7393,7 +7395,13 @@
   request(
     'POST',
     `/api/v2/tasks/${params.taskID}/runs/${params.runID}/retry`,
-    params,
+    {
+      ...params,
+      headers: {
+        ...params.headers,
+        'Content-Type': 'application/json; charset=utf-8',
+      },
+    },
     options
   ) as Promise<PostTasksRunsRetryResult>
```